roles/confluent.ssl/tasks/export_certs_from_keystore_and_truststore.yml

```
@@ -15,20 +16,23 @@
       | cut -d ":" -f2 \
       | cut -c2-
   register: keytool_type
+  no_log: true
 
 - name: Export Certificate from Keystore - PKCS12
   shell: |
     openssl pkcs12 -in {{keystore_path}} \
       -nokeys -out {{cert_path}} \
       -passin pass:{{keystore_storepass}}
   when: keytool_type.stdout == 'PKCS12'
+  no_log: true
```

Also wdyt about a variable?
```suggestion
  no_log: "{{mask_passwords_in_logs}}"
```

Or something like that, open to suggestions on the variable name.

Reason being when i debug ssl errors i like copying the shell command onto the host and running it.

---

nice idea, I used a variable named `mask_secrets` (default: `true`).